### ISSUE TITLE
Loinc 788-0:  "Erythrocyte distribution width [Ratio] by Automated count"

### ISSUE THREAD

**Issue by kingmanzhang** [obophenotype/human-phenotype-ontology#4212]
Suggest creating a new term for Loinc: 788-0 [Erythrocyte distribution width [Ratio] by Automated count] 
New term label: *Decreased RBC distribution width*
New term comment (if any): We are annotating low finding as *Anemia* but that might be a stretch 
Your biocurator ID for loinc2hpo (if desired): JGM:azhang

**Comment by pnrobinson** [comment]
This is definitely not anemia.
It is uncertain that low RDW is a medically relevant finding -- are we getting this from the asthma dataset?

**Comment by kingmanzhang** [comment]
It is the third most commonly conducted lab test in the asthma dataset, so shall we make sure it's correctly annotated?

**Comment by kingmanzhang** [comment]
@pnrobinson I just did a calculation. This LOINC test was performed 331,050 times in the Hush dataset, about half were normal (mapped to NOT *Abnormal erythrocyte morphology*), half were high (mapped to *Increased RBC distribution*), only 1835 times (0.5%) the finding was low. We did not annotate low as *Anemia* until today, so those low findings were mapped to no HPO term. I am going to delete the *Anemia* mapping now.
